app/controllers/editable_fields_controller.rb

```diff
@@ -0,0 +1,22 @@
+class EditableFieldsController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:update]
```

Comment:
```suggestion
   protect_from_forgery unless: -> { request.format.json? }, only: [:update]
```
https://api.rubyonrails.org/classes/ActionController/RequestForgeryProtection.html